codazen/webDiplomacy
# Feature/fbfw hold order

### Summary
This PR implements the **hold** move for the web diplomacy game. To execute a hold order the user must click on a unit and then click on the territory where that unit resides. 

Once the hold order has been set the user can click on the **save** button to save their moves. The following video shows how the hold order is implemented and verified. 

### Video
https://drive.google.com/file/d/108UgRTMv3wJNit-ys_MHu7GXm8DXh9hQ/view?usp=sharing

### Testing Plan
1. Spool up localhost game
2. Change units to move orders in the legacy application
3. Click on "Play Beta" to go to the react app
4. Execute a hold order by clicking on a unit and then clicking on the territory its on
5. Save the order by clicking the **save** or **ready** buttons. 
6. Go back to the legacy board and verify that the move orders are now hold orders.
